dev/MRTCore/packaging/native/Microsoft.ApplicationModel.Resources.WinRt.props

```diff
@@ -15,7 +15,7 @@
   <!-- OutputType Exe corresponds to C++. -->
   <ItemGroup Condition="'$(OutputType)' == 'Exe'">
     <Reference Include="Microsoft.ApplicationModel.Resources.winmd">
-      <HintPath>$(MSBuildThisFileDirectory)..\..\lib\uap10.0\Microsoft.ApplicationModel.Resources\Microsoft.ApplicationModel.Resources.winmd</HintPath>
```

Comment:
if you take a look at Microsoft.ProjectReunion.Foundation package, this is actually the current path. The reason to add an extra layer of folder is to avoid conflict with the file coming from WinUI.